### PR TITLE
Add Combine publishers

### PR DESCRIPTION
**Pull request by lickel**
Remove the Wrapper since it's duplicative.
I am tempted to remove the Pausable version as well, but that at least has some value at the moment.

### CODE REVIEW

**Comment by lickel** [comment]
Ignore the checks

File: FetchRequests/Sources/Controller/FetchedResultsControllerProtocol.swift [hunk 1]
```diff
@@ -7,12 +7,19 @@
...
 import Foundation
+import Combine
 
-internal protocol InternalFetchResultsControllerProtocol: AnyObject, FetchedResultsControllerProtocol {
+internal protocol InternalFetchResultsControllerProtocol: FetchedResultsControllerProtocol {
     func manuallyInsert(objects: [FetchedObject], emitChanges: Bool)
 }
 
-public protocol FetchedResultsControllerProtocol {
+public protocol DoublyObservableObject: ObservableObject {
```
Comment (AdamPro13) [inline FetchRequests/Sources/Controller/FetchedResultsControllerProtocol.swift:16]: Is `Doubly` a conventional name?

**Review by AdamPro13 — APPROVED** [review]

File: FetchRequests/Sources/Controller/FetchedResultsControllerProtocol.swift (hunk 1 above)
Comment (lickel) [reply FetchRequests/Sources/Controller/FetchedResultsControllerProtocol.swift:16]: I'm thinking doubly linked list

Comment (lickel) [reply FetchRequests/Sources/Controller/FetchedResultsControllerProtocol.swift:16]: I don't _really_ expect anyone to use the protocol, I just did it for organizational purposes

**Pull request merged**